src/ethereum_test_tools/spec/eof/eof_test.py

```diff
@@ -346,6 +346,9 @@ def generate(
         Generate the BlockchainTest fixture.
         """
         if fixture_format == FixtureFormats.EOF_TEST:
+            if self.deploy_tx and self.container_kind is None:
+                # Force validation in initcode mode
+                self.container_kind = ContainerKind.INITCODE
```

Comment:
Not sure this is good idea, maybe we want to keep these independent?

On the other hand there are already 2 different ways to specify EOF container kind for validation.